packages/table/src/useLeafygreenTable/useLeafygreenTable.types.ts

```diff
@@ -2,57 +2,48 @@ import { RefObject } from 'react';
 import { VirtualItem } from 'react-virtual';
 import { Cell, Row, Table, TableOptions } from '@tanstack/react-table';
 
-// Below is copied from react-virtual as their types are exported
+import { VirtualizerValues } from './reactVirtual.types';
 
-type ScrollAlignment = 'start' | 'center' | 'end' | 'auto';
-
-interface ScrollToOptions {
-  align: ScrollAlignment;
-}
-
-interface ScrollToOffsetOptions extends ScrollToOptions {}
-interface ScrollToIndexOptions extends ScrollToOptions {}
-
-export interface VirtualizerValues {
-  virtualItems: Array<VirtualItem>;
-  totalSize: number;
-  scrollToOffset: (index: number, options?: ScrollToOffsetOptions) => void;
-  scrollToIndex: (index: number, options?: ScrollToIndexOptions) => void;
-  measure: () => void;
-}
-
-// Above is copied from react-virtual as their types are not exported
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+export type LGRowData = {
+  [key: string]: any;
+};
```

Comment:
Should this extend from tanstack's `RowData` interface?

Reply:
Technically yes, but I was getting errors. I'll revert this later